### DISCUSSION TITLE
How i can select keymap for the vim-visual-multi action, because in my configuration <C-n> toggle nvim-tree

### DISCUSSION THREAD

**Discussion by ghost** [mg979/vim-visual-multi#286] [Q&A]
after adding vim-visual-multi to my lazy custom config, and after testing it.
it works good, but i notice that i can not open my nvim-tree because the keymap for vim-visual-multi it the same as the NvimTreeToggle and i dont want to change the nvim-tree toggler

**Comment by mslalith** [comment]
`vim.keymap.set("n", "<C-g>", "<Plug>(VM-Find-Under)")`
